packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-run-sample/main/java/testapp.md

```diff
@@ -19,13 +20,12 @@
 
 1. Execute `mvn` from the `...\okta-idx-java\samples\embedded-sign-in-widget\` sample app root directory.
 
-1. Navigate to the sample app's home page on your browser: `http://localhost:8080`. <br>The Embedded Sign In Widget + Spring Boot Example page appears.
+1. Navigate to the sample app's home page on your browser: `http://localhost:8080`. <br>The Embedded Sign-In Widget + Spring Boot Example page appears.
 
 1. Click **Next**. The Sign-In Widget appears.
-1. Enter the **Username** and **Password** for the user that you've created in
-   [Create your Okta account](/docs/guides/oie-embedded-common-org-setup/java/main/#create-your-okta-account).
+1. Enter the **Username** and **Password** for the user that you used to [create your Okta account](/docs/guides/oie-embedded-common-org-setup/java/main/#create-your-okta-account).
 
-1. Click **Sign in**. <br>After you've successfully signed in, the app redirects you to the Embedded Sign In Widget + Spring Boot Example home page.
+1. Click **Sign in**. <br>After you've successfully signed in, the app redirects you to the Embedded Sign-In Widget + Spring Boot Example home page.
```

Comment:
The "Click Next." step doesn't have a leading <br> tag. For consistency, maybe should remove them here too.